apis/clusters/v1beta1/cassandra_webhook.go

```diff
@@ -106,24 +106,10 @@ func (cv *cassandraValidator) ValidateCreate(ctx context.Context, obj runtime.Ob
 	}
 
 	for _, dc := range c.Spec.DataCentres {
-		err := dc.DataCentre.ValidateCreation()
+		err := cv.validateDataCenterCreation(c, dc)
```

Comment:
We use DataCentre not DataCenter name everywhere, so lets use it everywhere

Reply:
Renamed